cc/private/toolchain/BUILD.tpl

```diff
@@ -18,6 +18,7 @@ load(":cc_toolchain_config.bzl", "cc_toolchain_config")
 load(":armeabi_cc_toolchain_config.bzl", "armeabi_cc_toolchain_config")
 load("@rules_cc//cc/toolchains:cc_toolchain.bzl", "cc_toolchain")
 load("@rules_cc//cc/toolchains:cc_toolchain_suite.bzl", "cc_toolchain_suite")
+load("@bazel_tools//tools/cpp/modules_tools:wrapper.bzl", _generate_modmap_wrapper = "generate_modmap_wrapper")
```

Comment:
Please move this rule into rules_cc. The underlying binary can live in bazel_tools for now and will eventually move out.